packages/vite-app-ts/src/main.tsx

```diff
@@ -0,0 +1,104 @@
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { Main } from '@1hive/1hive-ui';
+
+import { useEthersContext } from 'eth-hooks/context';
+import { useDexEthPrice } from 'eth-hooks/dapps';
+import { asEthersAdaptor } from 'eth-hooks/functions';
+
+import { MainPageMenu } from './components/main';
+
+import { useBurnerFallback } from '~~/components/main/hooks/useBurnerFallback';
+import { useScaffoldProviders as useScaffoldAppProviders } from '~~/components/main/hooks/useScaffoldAppProviders';
+import { BURNER_FALLBACK_ENABLED } from '~~/config/appConfig';
+import { useConnectAppContracts, useLoadAppContracts } from '~~/config/contractContext';
+import { Account } from 'eth-components/ant';
+
+import { Header, MainWrapper, Sidebar, Content } from './main.styled';
+import Home from './pages/home';
+import History from './pages/history';
+import FaqView from './pages/faq';
+import Transactions from './pages/transactions';
+import { DollarOutlined, HomeOutlined, QuestionCircleOutlined, RetweetOutlined } from '@ant-design/icons';
+
+export const MainApp = () => {
+  const scaffoldAppProviders = useScaffoldAppProviders();
+  const ethersContext = useEthersContext();
+
+  useBurnerFallback(scaffoldAppProviders, BURNER_FALLBACK_ENABLED);
+
+  useLoadAppContracts();
+
+  useConnectAppContracts(asEthersAdaptor(ethersContext));
+
+  const [ethPrice] = useDexEthPrice(scaffoldAppProviders.mainnetAdaptor?.provider, scaffoldAppProviders.targetNetwork);
+
+  return (
+    <Main style={{ width: '100%' }}>
```

Comment:
@kamikazebr do you know how to make the page full size? with this component it makes centered, 
![Uploading Captura de ecrã 2022-06-25, às 18.28.52.png…]()